app/controllers/api/auth/stories_controller.rb

```diff
@@ -20,18 +19,14 @@ def check_user_network
     user_not_authorized unless current_user.networks.exists?(params[:network_id])
   end
 
-  def sorted(res)
-    res.order('updated_at desc')
```

Comment:
I think that may be a good default for the API.  When viewing all stories ... you get them in order of published date.  When viewing only your stories (including unpublished), you get them in order of updated_at.